src/index.js

```diff
@@ -1,8 +1,33 @@
-import React from 'react';
-import ReactDom from 'react-dom';
-import App from './components/App';
-
-ReactDom.render(
-    <App />,
-    document.getElementById("root")
-);
+const express = require('express');
+const mongoose = require('mongoose');
+
+const app = express();
+require('../react/node_modules/dotenv/lib/main').config();
+
+const username = process.env.MONGO_USERNAME;
+const password = process.env.MONGO_PWD;
+
+console.log(username);
+console.log(password);
+
+const db = mongoose.connect(`mongodb+srv://${username}:${password}@capmation-keeper.hpvo5.mongodb.net/test`, { useNewUrlParser: true, useUnifiedTopology: true });
```

Comment:
This connection string will establish a `test` database. It might be better to have a `capmation-keeper` database.

Reply:
ok